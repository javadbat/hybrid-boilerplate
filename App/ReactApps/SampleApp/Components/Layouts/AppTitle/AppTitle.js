import React from "react";
import generalConfig from "@config/GeneralConfig";

export const SampleAppTitle = () => (
  <div className="project-title">
    <h1>Sample ReactJs App</h1>
    <h5>mode: {generalConfig.env}</h5>
  </div>
);
