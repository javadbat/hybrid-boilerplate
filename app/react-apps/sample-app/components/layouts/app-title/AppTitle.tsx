import React from "react";
import generalConfig from "../../../../../../config/general-config";

export const SampleAppTitle = () => {
    return (
    <div className="project-title">
        <h1>Sample ReactJs App</h1>
        <h5>mode: {generalConfig.env}</h5>
    </div>
    )
}
   
