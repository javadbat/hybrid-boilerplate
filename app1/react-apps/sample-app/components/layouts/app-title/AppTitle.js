import React from "react";
import generalConfig from "@config/general-config.js";

export const SampleAppTitle = () => (
    <div className="project-title">
        <h1>Sample ReactJs App</h1>
        <h5>mode: {generalConfig.env}</h5>
    </div>
);
