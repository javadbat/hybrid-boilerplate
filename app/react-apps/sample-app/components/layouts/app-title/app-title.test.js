import renderer from 'react-test-renderer';
import {SampleAppTitle} from './AppTitle.tsx';
import {it,expect} from '@jest/globals';
import React from 'react';
it('component will render', () => {
  const component = renderer.create(
    <SampleAppTitle ></SampleAppTitle>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});