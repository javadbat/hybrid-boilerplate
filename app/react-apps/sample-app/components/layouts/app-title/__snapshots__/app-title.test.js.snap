// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component will render 1`] = `
<div
  className="project-title"
>
  <h1>
    Sample ReactJs App
  </h1>
  <h5>
    mode: 
    development
  </h5>
</div>
`;
